src/Filter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Interface Filter.
+ *
+ * @package Amp\AmpWP
+ */
+
+namespace Amp\AmpWP;
+
+/**
+ * Constants for the filters that the AmpWP plugin supports.
+ *
+ * @package Amp\AmpWP
+ */
+interface Filter {
+
+
+	const ENABLE_OPTIMIZER = 'amp_enable_optimizer';
+	const ENABLE_SSR       = 'amp_enable_ssr';
+	const OPTIMIZER_CONFIG = 'amp_optimizer_config';
```

Comment:
Note comments in `AMP_Theme_Support` where I think these string literals should be left inline with the `apply_filters` call for the sake of IDE static analysis.